pom.xml

```diff
@@ -147,33 +146,19 @@
 
     <properties>
 
-
-        <hbase1.artifactId.suffix>hbase1.x</hbase1.artifactId.suffix>
-        <hbase2.artifactId.suffix>hbase2.x</hbase2.artifactId.suffix>
-
-        <shims1.artifactId>omid-hbase-shims-${hbase1.artifactId.suffix}</shims1.artifactId>
-        <shims2.artifactId>omid-hbase-shims-${hbase2.artifactId.suffix}</shims2.artifactId>
-
-        <!-- hbase-1 profile props are here and not in profile section to work better with intelij-->
-        <shims.artifactId>${shims1.artifactId}</shims.artifactId>
-        <java.version>1.7</java.version>
-        <hadoop.version>${hadoop1.version}</hadoop.version>
-        <hbase.version>${hbase1.version}</hbase.version>
-        <shims.module>hbase-1</shims.module>
-        <hbase.artifactId.suffix>${hbase1.artifactId.suffix}</hbase.artifactId.suffix>
+        <java.version>1.8</java.version>
+        <commons-io.version>2.11.0</commons-io.version>
 
         <!-- Basic properties -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <!-- 3rd-Party Library Versioning -->
-        <hbase1.version>1.3.1</hbase1.version>
-        <hbase2.version>2.0.1</hbase2.version>
-        <hadoop1.version>2.7.5</hadoop1.version>
-        <hadoop2.version>3.0.0</hadoop2.version>
+        <hbase.version>2.4.12</hbase.version>
+        <hadoop.version>2.10.0</hadoop.version>
         <phoenix.thirdparty.version>2.0.0</phoenix.thirdparty.version>
         <guice.version>3.0</guice.version>
         <testng.version>6.10</testng.version>
-        <slf4j.version>1.7.7</slf4j.version>
+        <slf4j.version>1.7.30</slf4j.version>
```

Comment:
Let's go all the way up to slf4j.version 1.7.36, the latest, which transparently uses reload4j where necessary if anything slf4j dependency tries to pull in log4j 1.x.

Reply:
OK, I will update.
(Note that omid uses log4j2 already)

Reply:
This will mess the classpath up, as the slf4j backend pulls in relad4j, and we still pull log4j12.
I won't clean that up, as I plan to switch to log4j2 before 1.1 is released anayway.

Reply:
Done